lib/asynchronize.rb

```diff
@@ -1,14 +1,19 @@
 module Asynchronize
   require 'set'
   def self.included(base)
+      
+    # @@asynced_methods     ... methods we have already asynchronized.
+    # @@methods_to_async    ... methods that should be asynchronized.
+    # @@methods_asyncing    ... what is the short version of this?
+    #   Originally used a single value here, but that's not thread safe.
```

Comment:
Let's mention that this is referring to `@@methods_asyncing` if it's no longer paired with it.

Reply:
The reason I put `... what is the short version of this?` is because I don't know what sort of comment should go here :(